rs/sns/governance/CHANGELOG.md

```diff
@@ -10,6 +10,42 @@ here were moved from the adjacent `unreleased_changelog.md` file.
 
 INSERT NEW RELEASES HERE
 
+
+# 2025-02-03: Proposal 135067
+
+http://dashboard.internetcomputer.org/proposals/135067
+
+## Added
+
+* Enable SNSs to opt in for
+[automatically advancing its target version](https://forum.dfinity.org/t/proposal-opt-in-mechanism-for-automatic-sns-target-version-advancement/39874)
+to the newest version blessed by the NNS. To do so, please submit a `ManageNervousSystemParameters` 
+proposal, e.g.:
+
+    ```bash
+    dfx canister --network ic call $SNS_GOVERNANCE_CANISTER_ID manage_neuron '(
+    record {
+        subaccount = blob "'${PROPOSER_SNS_NEURON_SUBACCOUNT}'";
+        command = opt variant {
+        MakeProposal = record {
+            url = "https://forum.dfinity.org/t/proposal-opt-in-mechanism-for-automatic-sns-target-version-advancement/39874";
+            title = "Opt for automatic advancement of SNS target versions";
+            action = opt variant {
+            ManageNervousSystemParameters = record {
+                automatically_advance_target_version = opt true;
+            }
+            };
+            summary = "Enable automatically advancing the target version \
+                    of this SNS to speed up the delivery of SNS framework \
+                    upgrades that were already blessed by the NNS.";
+        }
+        };
+    },
+    )'
+    ```
+
+* Do not redact chunked Wasm data in `ProposalInfo` served from `SnsGov.list_proposals`.
```

Comment:
This is released Jan 27 (https://dashboard.internetcomputer.org/proposal/134989)

Reply:
That is the section it's under, no?